### ISSUE TITLE
Current date printing regardless of settings

### ISSUE THREAD

**Issue by jauslander** [mdjnelson/moodle-mod_customcert#556]
All of our custom certificates are currently printing the current date instead of any of the other settings. I tested all the options and they all print with today's date. The 'award date' that shows before viewing the certificate has the correct date.

**Comment by mdjnelson** [comment]
Is this just when repositioning the elements or when a student actually downloads the certificate? The repositioning page just uses today's date as the default.

**Comment by jauslander** [comment]
This is happening when students actually download the certificate.

**Comment by mdjnelson** [comment]
Can you please edit the date field and save it again?

**Comment by jauslander** [comment]
Yes, I just did and it is still printing the current date. I added each version of the date for testing purposes and they all display today's date.
